TrackingPerformance/Condor/condorJobs_sim.py

```diff
@@ -1,156 +1,248 @@
+    # =======================
+
+    # Create all possible combinations
+    import itertools
+
+    iter_of_combined_variables = itertools.product(
+        thetaList_, momentumList_, particleList_, DetectorModelList_
+    )
+
+    need_to_create_scripts = False
+
+    for theta, momentum, part, dect in iter_of_combined_variables:
+        for task_index in range(N_jobs_per_para_set):
+
+            output_file_name_parts = [
+                f"SIM_{dect}",
+                f"{part}",
+                f"{theta}_deg",
+                f"{momentum}_GeV",
+                f"{Nevt_per_job}_evts",
+                f"{task_index}",
+            ]
+            output_file_name = Path("_".join(output_file_name_parts)).with_suffix(
+                ".edm4hep.root"
+            )
 
+            # Check if the output file already exists and has correct Nb of events
+            output_dir = SIMEOSDir / part
+            output_dir.mkdir(parents=True, exist_ok=True)
+            output_file_path = output_dir / output_file_name
+            if CHECK_OUTPUT and output_file_path.exists():
+                root_file = ROOT.TFile(output_file_path, "READ")
+                events_tree = root_file.Get("events")
+                if events_tree:  # FIXME: why no else?
+                    if events_tree.GetEntries() == int(Nevt_per_job):
+                        root_file.Close()
+                        continue
+                root_file.Close()
+            else:
+                need_to_create_scripts = True
+
+            # if len(DetectorModelList_) != 1 or DetectorModelList_[0] != "ILD_l5_v11":
+            #     raise ValueError("so far only ILD_l5_v11 possible")
+
+            # Build ddsim command
+            arguments = [
+                # f" --compactFile /afs/cern.ch/work/g/gasadows/k4geo/FCCee/CLD/compact/{DetectorModelList_[0]}_3T/{DetectorModelList_[0]}.xml "
+                f" --compactFile $k4geo_DIR/{config.detModPaths['ILD_l5_v11']}",  # Note the change to use double quotes for the dictionary key
+                f"--outputFile {output_file_name}",
+                f"--steeringFile {config.sim_steering_file}",  # "CLDConfig/CLDConfig/cld_steer.py "
+                # TODO: FIX
+                # "--enableGun",
+                # f"--gun.particle {part}-",
+                # f"--gun.energy {momentum}*GeV",
+                # "--gun.distribution uniform",
+                # f"--gun.thetaMin {theta}*deg",
+                # f"--gun.thetaMax {theta}*deg",
+                # "--crossingAngleBoost 0",
+                # f"--numberOfEvents {Nevt_per_job}",
+            ]
+            command = f"ddsim {' '.join(arguments)} > /dev/null"
+
+            # Write bash script for job execution
+            bash_script = (
+                "#!/bin/bash \n"
+                f"source {environ_path} \n"
+                "git clone https://github.com/Victor-Schwan/TrackingStudies.git \n"  # FIXME: new repo location to config (to switch easily to local fork)
```

Comment:
Is this needed (or even correct)? Before we were cloning `CLDConfig` here to have the `cld_steer.py` file available.

Reply:
I only changed the Path. Maybe @gaswk can comment on this?

Reply:
You would have "only changed the path" if this was `github.com/gaswk/FullSim` but it was `github.com/gaswk/CLDConfig`

Reply:
If I understand correctly, this is superfluous. @gaswk you presumably cloned your fork from `CLDConfig` to `afs`, right? Then the job should be able to access it without having to clone it again if I am not mistaken:
"This has access to two shared file systems, AFS and EOS. The batch system can also see both filesystems."([batchdocs](https://batchdocs.web.cern.ch/concepts/dataflow.html))

Reply:
Yes, it has access to `afs` and can use your own fork from there. However, I think it is preferable to use the steering file from `https://github.com/key4hep/` by default. We could also make it configurable, allowing you to specify a different steering file if desired?

Reply:
In `cld_config` line 26, one can specify the steering file:
```
sim_steering_file = detectorDIR / "CHANGE" / "PATH"  # FIXME
```
The one from `https://github.com/key4hep/` should be accessible via `cvmfs`, shouldn't it?

Reply:
> The one from `https://github.com/key4hep/` should be accessible via `cvmfs`, shouldn't it?

It should be accessible in the future but not at the moment. The ILD equivalent [`ddsim_steer.py`](https://github.com/iLCSoft/ILDConfig/blob/master/StandardConfig/production/ddsim_steer.py) is also not available (yet) on cvmfs as far as I know...

Reply:
Technically the tags are available under `/cvmfs/ilc.desy.de/sw/ILDConfig/vxx-yy-zz/`, but not within a key4hep release yet.

Reply:
I removed the cloning. If you want another solution, I suggest to open a new issue